schema.sql

```diff
@@ -1,9 +0,0 @@
-CREATE TABLE animals(
-	id INT GENERATED BY DEFAULT AS IDENTITY,
```

Comment:
- [ ] You have created the file with the correct code but removed them and submitted the empty one. Insert these codes again and resubmit the PR. 👍